src/App.js

```diff
@@ -3,20 +3,42 @@ import {useState} from 'react'
 import countries from './countries'
 import getCountryData from './consumer'
 import geolocationStates from './geolocationStates'
-import {useGoogleCharts, useGeolocation, useChartUpdate, useResizeListener} from './hooks'
+import {useCountrySelectionStore, useGeolocation, useChartUpdate, useResizeListener, useGoogleChartSetUp} from './hooks'
```

Comment:
This time would probably be as good as any to run the js files through prettier :slightly_smiling_face:

Reply:
I fixed this line. Haven't run the file through a prettier tho. I don't think I have it set up. Feel free to do it if needed.

Reply:
I applied prettier